<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dispertech - Fiber Alignment</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="button_fiber_led">
         <property name="text">
          <string>LED</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Exposure</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="camera_exposure_line"/>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Gain</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="camera_gain_line"/>
       </item>
       <item>
        <widget class="QPushButton" name="save_core_button">
         <property name="text">
          <string>Save Core</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="save_laser_button">
         <property name="text">
          <string>Save Laser</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="fiber_core_position"/>
       </item>
       <item>
        <widget class="QLineEdit" name="laser_center_position"/>
       </item>
       <item>
        <widget class="QCheckBox" name="laser_track">
         <property name="text">
          <string>Laser</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="camera_widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Monitor the Fiber End</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_move_up">
   <property name="text">
    <string>Move Up</string>
   </property>
   <property name="toolTip">
    <string>Move mirror up</string>
   </property>
   <property name="shortcut">
    <string>Up</string>
   </property>
  </action>
  <action name="action_move_down">
   <property name="text">
    <string>Move Down</string>
   </property>
   <property name="shortcut">
    <string>Down</string>
   </property>
  </action>
  <action name="action_move_left">
   <property name="text">
    <string>Move Left</string>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
  </action>
  <action name="action_move_right">
   <property name="text">
    <string>Move Rigth</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
